client/landing/jetpack-cloud/sections/backups/rewind-flow/rewind-config-editor.tsx

```diff
@@ -58,6 +58,17 @@ const BackupRewindConfigEditor: FunctionComponent< Props > = ( {
 				},
 			} ),
 		},
+		{
+			name: 'roots',
+			label: translate(
+				'{{strong}}WordPress root{{/strong}} (includes wp-config php and any non WordPress files)',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( '{{strong}}WordPress root{{/strong}} (includes wp-config.php and any non-WordPress files)' )` *ES Score: 8*